app/views/customers/forgot_password.html.haml

```diff
@@ -3,7 +3,7 @@
 %ol
   %li Enter the email address you've registered with us below.
   %li Click "Reset My Password" and we'll email you a new password within 1 minute.
```

Comment:
Shouldn't this change to "and we'll email you a link to reset your password within 1 minute"?